src/domain/app/routes/AdminPageRoutes.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Redirect, Route, Switch } from 'react-router';
+
+import { ROUTES } from '../../../constants';
+import { Language } from '../../../types';
+import ApiPage from '../../help/pages/ApiPage';
+import ContactPage from '../../help/pages/ContactPage';
+import ControlPanelPage from '../../help/pages/ControlPanelPage';
+import FaqPage from '../../help/pages/FaqPage';
+import FeaturesPage from '../../help/pages/FeaturesPage';
+import GeneralInstructionsPage from '../../help/pages/GeneralInstructionsPage';
+import GeneralTechnologyPage from '../../help/pages/GeneralTechnologyPage';
+import ImageRightsPage from '../../help/pages/ImageRightsPage';
+import PlatformPage from '../../help/pages/PlatformPage';
+import SourceCodePage from '../../help/pages/SourceCodePage';
+import TermsOfUsePage from '../../help/pages/TermsOfUsePage';
+import NotFound from '.././../notFound/NotFound';
+const DocumentationPage = React.lazy(
+  () => import('../../help/pages/DocumentationPage')
+);
+
+interface Props {
+  locale: Language;
+}
+
+const InstructionsRoutes: React.FC<Props> = ({ locale }) => {
+  const getLocalePath = (path: string) => `/${locale}${path}`;
+
+  return (
+    <Switch>
+      <Redirect
+        exact
+        path={getLocalePath(ROUTES.INSTRUCTIONS)}
+        to={getLocalePath(ROUTES.INSTRUCTIONS_GENERAL)}
+      />
+      <Route
+        path={getLocalePath(ROUTES.INSTRUCTIONS_CONTROL_PANEL)}
+        component={ControlPanelPage}
+      />
+      <Route
+        path={getLocalePath(ROUTES.INSTRUCTIONS_FAQ)}
+        component={FaqPage}
+      />
+      <Route
+        path={getLocalePath(ROUTES.INSTRUCTIONS_GENERAL)}
+        component={GeneralInstructionsPage}
+      />
+      <Route
+        path={getLocalePath(ROUTES.INSTRUCTIONS_PLATFORM)}
+        component={PlatformPage}
+      />
+      <Route component={NotFound} />
+    </Switch>
+  );
+};
+
+const TechnologyRoutes: React.FC<Props> = ({ locale }) => {
+  const getLocalePath = (path: string) => `/${locale}${path}`;
+
+  return (
+    <React.Suspense fallback={<div />}>
+      <Switch>
+        <Redirect
+          exact
+          path={getLocalePath(ROUTES.TECHNOLOGY)}
+          to={getLocalePath(ROUTES.TECHNOLOGY_GENERAL)}
+        />
+        <Route
+          path={getLocalePath(ROUTES.TECHNOLOGY_API)}
+          component={ApiPage}
+        />
+        <Route
+          path={getLocalePath(ROUTES.TECHNOLOGY_DOCUMENTATION)}
```

Comment:
_Disclaimer: I'm not familiar with TypeScript at all._

I noticed that the AdminPageRoutes.tsx returns are re-using the same Switch statements with tons of Route's without actually changing the data that much from a logical perspective (lots of repetition). I therefore propose that the code should be refactored using a single function call for each return with a single Switch within. The Routes can be appended to itself (if possible somehow) using recursion or a simple loop from mapped lists of specified ROUTE paths and their components. If possible the component names could be auto-formatted from the ROUTES attribute names so you don't even necessarily need to make them into key value pairs, the function could do that as well.

Of course, all of what I proposed needs internal input from the teams front-end developers.

Lines:
29, 60, 97, 120.